demo-project/.pre-commit-config.yaml

```diff
@@ -14,31 +14,11 @@ repos:
           - id: check-json # Checks json files for parseable syntax.
           - id: check-case-conflict # Check for files that would conflict in case-insensitive filesystems
           - id: check-merge-conflict # Check for files that contain merge conflict strings.
-          - id: debug-statements # Check for debugger imports and py37+ `breakpoint()` calls in python source.
```

Comment:
`ruff`'s `T100` ruleset, which I'm enabling, checks and autofixes that, so this pre-commit hook becomes redundant.